ruby/range-cover-vs-include.md

```diff
@@ -0,0 +1,18 @@
+# Range cover? vs include? method
+
+The `cover?` method yields better performance than `include?`,
+however there are subtle differences to when one should be used over other.
```

Comment:
I found one of the benchmarks for these here 
https://gist.github.com/inkel/6090991

Results:
                             user     system      total        real
#include? - best case    0.130000   0.000000   0.130000 (  0.129843)
#include? - worst case   0.130000   0.000000   0.130000 (  0.130545)
#include? - avg. case    0.130000   0.000000   0.130000 (  0.127180)
#cover? - best case      0.130000   0.000000   0.130000 (  0.127623)
#cover? - worst case     0.130000   0.000000   0.130000 (  0.127207)
#cover? - avg. case      0.120000   0.000000   0.120000 (  0.127252)